app/src/androidTest/java/com/github/sdpcoachme/ScheduleActivityTest.kt

```diff
@@ -0,0 +1,149 @@
+            description = "Tune in to find out about how we're furthering our mission to organize the world’s information and make it universally accessible and useful.",
+        ),
+        Event(
+            name = "Developer Keynote",
+            color = Color(0xFFAFBBF2).value.toString(),
+            start = currentMonday.plusDays(2).atTime(7, 0, 0).toString(),
+            end = currentMonday.plusDays(2).atTime(9, 0, 0).toString(),
+            description = "Learn about the latest updates to our developer products and platforms from Google Developers.",
+        ),
+        Event(
+            name = "What's new in Android",
+            color = Color(0xFF1B998B).value.toString(),
+            start = currentMonday.plusDays(2).atTime(10, 0, 0).toString(),
+            end = currentMonday.plusDays(2).atTime(12, 0, 0).toString(),
+            description = "In this Keynote, Chet Haase, Dan Sandler, and Romain Guy discuss the latest Android features and enhancements for developers.",
+        ),
+        Event(
+            name = "What's new in Machine Learning",
+            color = Color(0xFFF4BFDB).value.toString(),
+            start = currentMonday.plusDays(2).atTime(22, 0, 0).toString(),
+            end = currentMonday.plusDays(3).atTime(4, 0, 0).toString(),
+            description = "Learn about the latest and greatest in ML from Google. We’ll cover what’s available to developers when it comes to creating, understanding, and deploying models for a variety of different applications.",
+        ),
+        Event(
+            name = "What's new in Material Design",
+            color = Color(0xFF6DD3CE).value.toString(),
+            start = currentMonday.plusDays(3).atTime(13, 0, 0).toString(),
+            end = currentMonday.plusDays(3).atTime(15, 0, 0).toString(),
+            description = "Learn about the latest design improvements to help you build personal dynamic experiences with Material Design.",
+        ),
+        Event(
+            name = "Jetpack Compose Basics",
+            color = Color(0xFF1B998B).value.toString(),
+            start = currentMonday.plusDays(4).atTime(9, 0, 0).toString(),
+            end = currentMonday.plusDays(4).atTime(13, 0, 0).toString(),
+            description = "This Workshop will take you through the basics of building your first app with Jetpack Compose, Android's new modern UI toolkit that simplifies and accelerates UI development on Android.",
+        ),
+    )
+
+    @get:Rule
+    val composeTestRule = createEmptyComposeRule()
+
+    @Test
+    fun addEventsToDatabaseUpdatesUserInfoCorrectly() {
+        defaultEmail
+        val oldUserInfo = database.getUser(defaultEmail)
+
+        database.addEventsToDatabase(defaultEmail, eventList)
+
+        val newUserInfo = database.getUser(defaultEmail)
+        newUserInfo.thenAccept {
+            assert(oldUserInfo != newUserInfo)
+            assert(it.events == eventList)
```

Comment:
Maybe here using the assertTrue method of JUnit would be better.

In general, JUnit assertion methods provide more context when tests fail compared to kotlin buit-in assertions.